sdk/helper/ldaputil/config.go

```diff
@@ -256,7 +255,7 @@ Default: ({{.UserAttr}}={{.Username}})`,
 		"max_page_size": {
 			Type:        framework.TypeInt,
 			Description: "The maximum number of results to return for a single paged query. If not set, the server default will be used for paged searches. A requested max_page_size of 0 is interpreted as no limit by LDAP servers. If set to a negative value, search requests will not be paged.",
```

Comment:
This description could use a re-work.

> If not set, the server default will be used for paged searches.

No longer true, if the user does not set this, searches will be done without paging.

> A requested max_page_size of 0 is interpreted as no limit by LDAP servers.

Not true. A size of 0 is invalid and causes the LDAP server to return no results.

Reply:
Go LDAP doc suggests otherwise: https://github.com/go-ldap/ldap/blob/v2.5.1/search.go#L305

Reply:
(See response above)